presto-main/src/test/java/io/prestosql/sql/planner/TestDynamicFilter.java

```diff
@@ -142,6 +142,38 @@ public void testEmptyJoinCriteria()
                                         tableScan("lineitem")))));
     }
 
+    @Test
+    public void testCrossJoinInequalityDF()
+    {
+        assertPlan("SELECT o.orderkey FROM orders o, lineitem l WHERE o.orderkey < l.orderkey",
+                anyTree(filter("O_ORDERKEY < L_ORDERKEY",
+                        join(
+                                INNER,
+                                ImmutableList.of(),
+                                ImmutableMap.of("O_ORDERKEY", "L_ORDERKEY"),
```

Comment:
Please extend matcher so that we can match ineq type too

Reply:
Added `DynamicFilterPattern` to match inequality DFs.